portal/code/cx-portal/src/state/features/userAdministration/userAdministrationSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { RootState } from 'state/store'
+import { UserAdministrationInitialState } from 'types/userAdministration/UserAdministrationTypes'
+import { fetchTenantUsers } from './userAdministrationActions'
+
+const initialState: UserAdministrationInitialState = {
+  tenantUsers: [],
+  loading: true,
+  error: ''
+}
+
+const userAdministrationSlice = createSlice({
+  name: 'userAdministration',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(fetchTenantUsers.pending, (state) => {
```

Comment:
Probably `state.error` should get reset here as well

Reply:
done - reset all state values for all cases